core/pkg/hostsensorutils/hostsensorgetfrompod.go

```diff
@@ -99,6 +100,28 @@ func (hsh *HostSensorHandler) sendAllPodsHTTPGETRequest(path, requestKind string
 	return res, nil
 }
 
+// return host-scanner version
+func (hsh *HostSensorHandler) GetVersion() ([]byte, error) {
```

Comment:
It should concern us IMO because if a user has host-scanner running in their cluster for some reason (applied manually/ previous KS scan stopped in the middle) Kubescape will use it instead of applying a new one. This causes the case that even if a user updates KS version they might still be running against an old version of host-scanner, and probably not aware of this.
Maybe we should change KS behavior to run a new host-scanner by default and allow users to override with a flag or something. (?)
I'm a little less worried about a different image because I'm assuming if a user knowingly changed the image they'll be aware of that but I can add it too if you think it's important.